pyproject.toml

```diff
@@ -1,17 +1,30 @@
-[tool.poetry]
+[project]
+authors = [{ name = "Justin Mayer", email = "authors@getpelican.com" }]
+license = { text = "AGPLv3" }
+requires-python = ">=3.8,<4.0"
+dependencies = [
+    "blinker>=1.4",
+    "docutils>=0.16",
+    "feedgenerator>=1.9",
+    "jinja2>=2.7",
+    "pygments>=2.6",
+    "python-dateutil>=2.8",
+    "rich>=10.1",
+    "unidecode>=1.1",
+    "backports-zoneinfo<1.0.0,>=0.2.1; python_version < \"3.9\"",
```

Comment:
Is that escaping required? Could it not be `python_version<3.9` instead?

Reply:
no one need either escaping or `"...< '3.9'"`